torch/_inductor/codegen/triton_utils.py

```diff
@@ -119,6 +121,7 @@ def is_aligned(x: KernelArgType, alignment: int, include_tensor: bool) -> bool:
         for i, arg in zip(indices, args)
         if isinstance(arg, SizeArg)
         and arg.expr is not None
+        and isinstance(arg.expr, (int, sympy.Expr))
```

Comment:
Why did you also include `sympy.Expr`? not symint?

unit test does not cover this case

Reply:
If I don't include sympy.Expr, symbols statically know to be equal to 1 won't be considered. I assume that the next line

```
and V.graph.sizevars.statically_known_equals(arg.expr, 1)
```

will filter out non-1 `sympy.Expr`s (including 1.0). @ezyang could you confirm if this is a valid assumption? Or should I replace with `sympy.Integer`?

Reply:
So, I agree that we won't have SymInts at this point in time, in Inductor generally you're only ever dealing with int/sympy.Expr

That being said, I don't really know what the type invariants on SizeArg are, so I don't really know how to review this. Maybe I'll be satisfied with a comment here explaining the situation.

Reply:
Added a comment.

Reply:
@ezyang does this look good to you now? Thanks!